index.hbs

```diff
@@ -59,9 +59,12 @@ into the {body} of the default.hbs template --}}
                           <span>{{t "Featured"}}</span>
                         </div>
                         <a href="{{url}}" class="m-featured-article__content">
-                          <h1 class="m-featured-article__title js-featured-article-title" title="{{title}}">
+                          <h2 class="m-featured-article__title js-featured-article-title" title="{{title}}">
                             {{title}}
-                          </h1>
+                          </h2>
+                          {{#if meta_description}}
```

Comment:
The goal of this theme is to be as clean as possible, that's why I don't show the description or the excerpt of posts, if you want to show it you can do it on your own instance of course but I'm afraid that it's something that I won't implement so please remove this part from this PR.

Reply:
I think that you are perfectly right. I also like it clean. SEO made me do it :)
Please consider the switch from H1 to H2.